pkg/v1/providers/ytt/02_addons/management-packages/featuregates/add_featuregates_package.yaml

```diff
@@ -161,7 +161,7 @@ stringData:
     namespace: tkg-system
 #@ end
 
-#@ if data.values.TKG_CLUSTER_ROLE != "workload" and data.values.PROVIDER_TYPE != "tkg-service-vsphere":
+#@ if data.values.TKG_CLUSTER_ROLE != "workload" and data.values.PROVIDER_TYPE != "tkg-service-vsphere"  and not data.values.FEATURE_FLAG_PACKAGE_BASED_LCM:
```

Comment:
Is the intention to exclude this overlay from the upgrade case?

Reply:
No. I am excluding this at the moment because it applies the same `PackageInstall` again using CRS that we have already applied as part of TKG composite package installation. Basically if `FEATURE_FLAG_PACKAGE_BASED_LCM` is enabled we do not need to reinstall this package again using CRS or during cluster upgrade.